package.json

```diff
@@ -66,6 +67,7 @@
     "rollup": "^2.73.0",
     "rollup-plugin-summary": "^1.4.3",
     "rollup-plugin-terser": "^7.0.2",
+    "ts-prune": "^0.10.3",
```

Comment:
just a heads-up that ts-prune is going into [maintenance mode](https://github.com/nadeesha/ts-prune?tab=readme-ov-file#-ts-prune-is-going-into-maintanence-mode). We might want to explore alternatives in the future.

Reply:
Change to use `knip` recommended in ts-prune repo